File: 인터넷 강의/python_ruby/Class_member/3.py
class Cs:

    # ! 클래스 변수는 이런식으로 선언함!
    count = 0

    def __init__(self):
        Cs.count = Cs.count + 1

    @classmethod
    def getCount(cls):
        print(cls)
        return Cs.count

i1 = Cs()
i2 = Cs()
i3 = Cs()

print(Cs.getCount())

File: 인터넷 강의/python_ruby/Class_member/3.rb
class Cs
    
    @@count = 0

    def initialize()
        # ! class 에 속한 변수는 "@"를 두개 붙이면 된다.
        @@count = @@count + 1
    end

    def Cs.getCount()
        @@count
    end

end

i1 = Cs.new()
i2 = Cs.new()
i3 = Cs.new()

p Cs.getCount()